### DISCUSSION TITLE
Is this debuggable?

### DISCUSSION THREAD

**Discussion by jazzcake** [gilzoide/godot-lua-pluginscript#43] [Q&A] [answered]
Hello, I'd like to know it is possible to set break pointer in godo script editor.

**Comment by gilzoide** [comment, marked as answer]
Unfortuantely, no, it is not possible =/

You can, however, use something like [debugger.lua](https://github.com/slembcke/debugger.lua) and run Godot from the terminal to hit break points manually set in your code.
There might be other solutions for debugging Lua, but this is the only one I've ever used.
